crates/ibc/src/lib.rs

```diff
@@ -53,6 +53,9 @@ pub mod clients;
 pub mod core;
 pub mod hosts;
 
+#[cfg(feature = "grpc")]
+pub mod services;
+
 #[cfg(any(test, feature = "mocks"))]
 pub mod mock;
```

Comment:
In a later PR (Would you open an issue?), nice to implement gRPC context for `MockContext` and include corresponding tests to do queries and match results.

Reply:
Agreed. I will open a PR.